spec/lib/api/v3/work_packages/schema/work_package_schema_spec.rb

```diff
@@ -37,28 +37,15 @@
                       type: type)
   }
 
-  shared_examples_for 'WorkPackageSchema#available_custom_fields' do
-    let(:cf1) { double }
-    let(:cf2) { double }
-    let(:cf3) { double }
-
-    before do
-      scope_double = double
-      allow(scope_double).to receive(:all).and_return([cf1, cf2])
-      allow(type).to receive(:custom_fields).and_return(scope_double)
-      allow(project).to receive(:all_work_package_custom_fields).and_return([cf2, cf3])
-    end
-
-    it 'is expected to return custom fields available in project AND type' do
-      expect(subject.available_custom_fields).to eql([cf2])
-    end
-  end
-
   context 'created from work package' do
     subject { described_class.new(work_package: work_package) }
 
-    it 'defines assignable values' do
-      expect(subject.defines_assignable_values?).to be_true
+    it 'has the project set' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.